src/test/java/testsuite/integration/AwsIamAuthenticationIntegrationTest.java

```diff
@@ -38,7 +38,7 @@
 import java.sql.SQLException;
 import java.util.Properties;
 
-@Disabled
+//@Disabled
```

Comment:
Why not just removed this `@Disabled` flag instead of commenting it out?

Reply:
We should disable this test.